src/Evolution/Systems/ScalarTensor/Tags.hpp

```diff
@@ -0,0 +1,22 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace ScalarTensor::Tags {
+
+/*!
+ * \brief Represents the trace-reversed stress-energy tensor of the scalar
+ * field.
+ */
+struct TraceReversedStressEnergy : db::SimpleTag {
```

Comment:
Please add some docs in the `ScalarTensor` namespace to clarify where you need this